docs/quickstart-cli.md

```diff
@@ -131,29 +139,26 @@ wget --no-check-certificate `kubectl get machineregistration -n fleet-default my
 ```
 :::
 
-Now this is the last step, you need to prepare an Elemental Teal seed image that includes automatically (not for aarch64 yet) the initial registration config, so
-it can be auto registered, installed and fully deployed as part of your cluster. The contents of the file are nothing 
-more than the registration URL that the node needs to register and the proper server certificate, so it can connect securely.
+The contents of the registration config file are nothing  more than the registration URL that the node needs to register, the proper server certificate and few options for the registration process, so it can connect securely.
```

Comment:
```suggestion
The contents of the registration config file are nothing more than the registration URL that the node needs to register, the proper server certificate and few options for the registration process, so it can connect securely.
```